src/runtime_src/core/edge/user/system_linux.cpp

```diff
@@ -44,6 +44,10 @@
 #include "shim/xdna_device.h"
 #endif
 
+#ifdef XDNA_VE2 // this is temporary flag , will be removed once "shim_ve2" is moved under XDNA/src
+#include "shim_ve2/xdna_device.h"
+#endif
```

Comment:
As discussed offline, 
Remove EDGE_VE2 flag & add EDGE_VE2_XDNA as else of XDNA_VE2.
Once we move to public xdna, we should cleanup these flags.

Reply:
> As discussed offline, Remove EDGE_VE2 flag & add EDGE_VE2_XDNA as else of XDNA_VE2. Once we move to public xdna, we should cleanup these flags.

Hi @chvamshi-xilinx ,

Are you talking about removing the flag but leave the code here in XRT? Or do you plan to move entire ve2 shim to xdna-driver repo later? @sonals , @stsoe and me discussed offline and it seems that xdna-driver repo would be a better place for ve2 shim code. Can we just do it now instead of leaving them in XRT?

Reply:
@chvamshi-xilinx .  I don't understand the intent here?  XRT referencing shim specific code is not right.  There should not be any references to xdna, nor any inclusion of xdna repo code.